### PR TITLE
Feature/plant care reminders

### PR DESCRIPTION
**Pull request by Fabi911**

### CODE REVIEW

**Review by onemanwenttomow — CHANGES_REQUESTED** [review]
Nice job team 🎉

I added a few thoughts and suggestions.

File: pages/reminder.js [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+      monthName = "October";
+      break;
+    case 10:
+      monthName = "November";
+      break;
+    case 11:
+      monthName = "December";
+      break;
+    default:
+      monthName = "";
```
[the remark this answers is not in the document]
Comment (Alphawave1989) [reply pages/reminder.js]: implemented suggsetions

File: components/Form.js [hunk 2]
```diff
@@ -13,15 +12,51 @@ const StyledForm = styled.form`
...
+    //   document
+    //     .querySelectorAll('input[type="checkbox"]:checked')
+    //     .forEach((checkbox) => {
+    //       fertiliserSeason.push(checkbox.value);
+    //     });
+
+    //   data.fertiliser_season = fertiliserSeason; //save checkbox array in object
+    //   onSubmit(data);
+    //   entrySucessful();
+    // }
```
Comment (onemanwenttomow) [inline components/Form.js:50]: We should remove this comment before merging to main. If you want to find the code again you can always look through previous commits. i.e. [here](https://github.com/Fabi911/Capstone_Plant_Care/blob/4cac275395ecfa827403ca35e60440ce72d2c9ce/components/Form.js)

**Review by onemanwenttomow — APPROVED** [review]
One comment could be removed otherwise LGTM 🎉

**Pull request merged**